src/configure.js

```diff
@@ -136,6 +151,18 @@ const configureDefaults = async function() {
 const configure = async function( configuration ) {
     const sitesPath = path.resolve( configuration.sitesPath );
     const snapshotsPath = path.resolve( configuration.snapshotsPath );
+    const globalServicesPath = path.join( getConfigDirectory(), 'global' );
+
+    if ( configuration.overwriteGlobal ) {
+        try {
+            await fs.ensureDir( globalServicesPath );
+            await fs.copy( globalPath, path.join( getConfigDirectory(), 'global' ) );
+        } catch ( ex ) {
+            console.log( ex );
```

Comment:
@dustinrue let's add it to the following `console.error` call to make sure that we output errors to the STDERR stream only.